.github/workflows/build-and-test.yaml

```diff
@@ -0,0 +1,38 @@
+name: Build and Test
```

Comment:
CI is better, as it does more than build and test. Same for the filename (ci.yaml).